dotnet-logging/LogLevelsWithSerilog/Tests/LogLevelsWithSerilogTests.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net6.0</TargetFramework>
+		<TargetFramework>net7.0</TargetFramework>
+		<LangVersion>10</LangVersion>
```

Comment:
This line is unnecessary since .NET 7.0 defaults to C# 11

Reply:
updated